editors/vscode/src/common.ts

```diff
@@ -181,28 +210,28 @@ export class PropertiesViewProvider implements vscode.WebviewViewProvider {
                 command: "show_welcome",
                 message: "The active editor does not contain a Slint file.",
             });
+            refresh(); // Language Information might not be available yet!
```

Comment:
This place makes me wonder: Instead of continuing to poll every second when editing say a `.rs` file (or generally, a file from a different language), can't we wait until we receive notification that the current file matches the slint language?